src/ansys/fluent/core/session_solver.pyi

```diff
@@ -34,22 +21,4 @@ class Solver:
     def read_case(self, file_name: str): ...
     def write_case(self, file_name: str): ...
     @property
-    def file(self) -> file: ...
-    @property
-    def mesh(self) -> mesh: ...
-    @property
-    def server(self) -> server: ...
-    @property
-    def setup(self) -> setup: ...
-    @property
-    def solution(self) -> solution: ...
-    @property
-    def results(self) -> results: ...
-    @property
-    def parametric_studies(self) -> parametric_studies: ...
-    @property
-    def current_parametric_study(self) -> current_parametric_study: ...
-    @property
     def parameters(self) -> parameters: ...
```

Comment:
This won't also be exposed at top level, I think.

Reply:
Done, Thank you.